src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -536,7 +561,9 @@ mod tests {
         let block_1_second_key = iter.key().to_vec();
         // Use a big enough seek key and result in invalid iterator.
         let seek_key = test_key_of(30001);
-        iter.seek_idx(0, Some(seek_key.to_ref())).await.unwrap();
+        iter.seek_idx(table_infos.len() - 1, Some(seek_key.to_ref()))
```

Comment:
why do we change this test case to seek to the end?

Reply:
> why do we change this test case to seek to the end?

Because the original test is not a use case and will never happen.
Actually, this seek key `test_key_of(30001)` is larger than all blocks.